lighthouse-core/report/html/renderer/details-renderer.js

```diff
@@ -20,6 +20,14 @@
 
 /** @typedef {import('./dom.js')} DOM */
 
+// Convenience types for localized AuditDetails.
+/** @typedef {LH.FormattedIcu<LH.Audit.Details>} AuditDetails */
+/** @typedef {LH.FormattedIcu<LH.Audit.Details.ItemValue>} TableItemValue */
+/** @typedef {LH.FormattedIcu<LH.Audit.Details.Opportunity>} OpportunityTable */
+/** @typedef {LH.FormattedIcu<LH.Audit.Details.OpportunityColumnHeading>} OpportunityTableHeading */
+/** @typedef {LH.FormattedIcu<LH.Audit.Details.Table>} Table */
+/** @typedef {LH.FormattedIcu<LH.Audit.Details.TableColumnHeading>} TableHeading */
```

Comment:
This is my least favorite part of this change and I haven't come up with a nice solution yet. Unlike with Audit product/result, we don't have a good split on audit details (the same types are used in audits and in the LHR).

The types are all correct on the LHR, but if you want to refer to the type in isolation (and don't want to do `LH.Result['audits'][number]['details']['whatever']['etc']['etc']`) you need a reference to the type directly like this.

Other options:
- we have some sort of audit details product/result split?
- move these under `LH.Result` namespace?

Reply:
> move these under LH.Result namespace?

That seems reasonable but this works too.